battlebots/database/access.py

```diff
@@ -37,7 +37,8 @@ def remove_bot(user, botname):
                         % (user.nickname, botname, code_dir))
         pass
 
-    bot = session.query(Bot).filter_by(user=user, name=botname).one_or_none()
```

Comment:
That last wouldnt close that session connection then?
On 8 Apr 2016 11:00, "Stijn Seghers" <notifications@github.com> wrote:

> In battlebots/database/access.py
> <https://github.com/ZeusWPI/aichallenge/pull/85#discussion_r58997827>:
>
> > @@ -37,7 +37,8 @@ def remove_bot(user, botname):
> >                          % (user.nickname, botname, code_dir))
> >          pass
> >
> > -    bot = session.query(Bot).filter_by(user=user, name=botname).one_or_none()
>
> The commit/rollback part would indeed be unnecessary, so maybe we can make
> two kinds of context managers? Or maybe change Session.query(...) to
> Session().query(...)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ZeusWPI/aichallenge/pull/85/files/1478b56574b04b5a61ef6433eb7d796aa0f7b628#r58997827>
>